index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
```

Comment:
- [ ] I get some W3C validator. Kindly check that.

> screenshot
![image](https://user-images.githubusercontent.com/57414789/82617109-b73e4d00-9b94-11ea-813f-33ab284f654f.png)